### ISSUE TITLE
Italic Support for Geist Mono

### ISSUE THREAD

**Issue by Codevka** [vercel/geist-font#29]
**Font Name (Geist Sans/Geist Mono):**
- [x] Geist Sans
- [x] Geist Mono

**Description of the Issue:**
Geist Mono currently does not support italics, a feature commonly found in many code editor themes. I suggest considering the addition of italics support to Geist Mono, which could contribute to an even more enhanced coding experience for users.

**Comment by luciascarlet** [comment]
I agree, but I believe Geist Sans should also be expanded with an italic version in the future. Italics often come up in long-form text, and if not present in a font, most environments will fall back to synthetic italics which won't be consistent across operating systems and really just do not look good.

**Comment by AdamBD** [comment]
is this on the roadmap?

**Comment by augiethehuman** [comment]
I don't think so, there hasn't been any update.

**Comment by mrclayman** [comment]
> I don't think so, there hasn't been any update.

I hope the team would reconsider. The italics don't have to be crazy. Just something with a little nuance to give them their own touch of personality.

**Comment by luciascarlet** [comment]
While I have a thing for fancy cursive italics myself I'd be totally fine with a simple oblique; the issue in my opinion is moreso that synthetic italics just look *hideous*. Geist is therefore essentially unusable in any environment where italics are required, which is a shame really.

**Comment by mwolf1989** [comment]
Hey is there any news on this?

**Comment by Pukimaa** [comment]
Don't think so. Our team just made a synthetic version, but obviously a native italic font would be much needed 🙏

**Comment by ara303** [comment]
I don't want to "me too!" on this but the lack of italics is the only reason I can't use Geist literally everywhere. I love the font but _need_ italics (😏). Oblique would be fine.

**Comment by augiethehuman** [comment]
Literally.

**Comment by sprouvy** [comment]
Up. #BringItalicTogether

**Comment by carlesandres** [comment]
I would *love* to see Geist in italics!

**Comment by MGSimard** [comment]
For a font claiming to be the be all end all _"font for developers and designers"_, the fact that it lacks basic versatility like italics is a little ironic. Guess we'll have to stick with something else for actually useful typography.

**Comment by ravshansbox** [comment]
It would be great to have a real italic (like the one IBM Plex Mono has) rather than an oblique version.

**Comment by FlawlessCasual17** [comment]
Hey, why has this been delayed? It has been 2 years now. What is **Vercel**'s reasoning for this?

**Comment by guidoferreyra** [comment]
There is a [pre release](https://github.com/vercel/geist-font/releases/tag/1.5.0-alpha) including Italics for Geist and the Mono italics are being developed at this moment.

**Comment by guidoferreyra** [comment]
Version 1.5 has been released adding italics to Geist and Geist Mono :)
